### PR TITLE
Add latitude / longitude position for listing details map

### PR DESCRIPTION
**Pull request by henryhu1**

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt [hunk 1]
```diff
@@ -418,11 +424,12 @@ fun GoogleMapComposable(
...
 	cameraPositionState: CameraPositionState,
 	disableColumnScrolling: () -> Unit,
-	position: LatLng?,
+	position: LatLng,
 ) {
 	GoogleMap(
 		modifier = modifier
-			.size(dimensionResource(id = R.dimen.listing_details_image))
+			.fillMaxWidth(fraction = 1.0f)
+			.aspectRatio(ratio = 1.0f)
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt:432]: Looks familiar for some reason lol

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt [hunk 2]
```diff
@@ -437,17 +444,18 @@ fun GoogleMapComposable(
 			),
 		cameraPositionState = cameraPositionState
 	) {
-		Marker(
-			state = MarkerState(position = position ?: SINGAPORE),
+		Circle(
+			center = position,
+			radius = 500.0,
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt]: Maybe just put this in a constant.

File: android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrColors.kt [hunk 3]
```diff
@@ -123,6 +123,8 @@ private val textFieldBorderColorDark = Color((0xFFE5E5E5))
 
 val Purple900 = Color(0xFF4A148C)
 val Purple500 = Color(0xFF9C27B0)
+val mapCircleFill = Color(0x8042A5F5)
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrColors.kt:126]: Are the maps dark-mode conscious?

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

File: android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrColors.kt (hunk 3 above)
Comment (henryhu1) [reply android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrColors.kt:126]: nope... I think the blue will look good for both light and dark mode

**Review by andr-dev — APPROVED** [review]
Nice work!

**Pull request merged**